From 3159552236718e263578e250cfdcd9cc1140f6ea Mon Sep 17 00:00:00 2001
From: Vinu Vaghasia <vinu.vaghasia@amd.com>
Date: Fri, 4 Feb 2022 10:09:44 -0600
Subject: [PATCH 1/1] phosphor-network: Modify removed previous changes for 2nd
 MAC address display

Prev change had a side effect with MAC address display on the Web GUI.
It was displaying only one MAC addr for eth0 interface, was missing for
eth1.
---
 ethernet_interface.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/ethernet_interface.cpp b/ethernet_interface.cpp
index 9928f38..e9b98de 100644
--- a/ethernet_interface.cpp
+++ b/ethernet_interface.cpp
@@ -115,10 +115,7 @@ EthernetInterface::EthernetInterface(sdbusplus::bus::bus& bus,
     // would be same as parent interface.
     if (intfName.find(".") == std::string::npos)
     {
-        if (intfName.compare("eth0") == 0)
-        {
             activeMACAddress = MacAddressIntf::macAddress(getMACAddress(intfName));
-        }
     }
     EthernetInterfaceIntf::ntpServers(getNTPServersFromConf());

--
2.17.1

